packages/one-app-dev-bundler/package.json

```diff
@@ -39,6 +39,7 @@
     "esbuild-plugin-svgr": "^1.0.1",
     "filesize": "^9.0.1",
     "glob-all": "^3.3.0",
+    "lodash.snakecase": "^4.1.1",
```

Comment:
Lets not get into using lodash, can we implement this ourselves?

Reply:
I agree on avoid lodash when possible, but, in this case, I'm using `snakecase` because, after looking at the implementation, it's pretty small, it relies mainly on `words` (https://github.com/lodash/lodash/blob/master/snakeCase.js#L1) which checks if it has unicode work or ascii words and properly deals with them (https://github.com/lodash/lodash/blob/master/words.js) which is a stronger implementation than just simply doing something like `str.split(' ')`

Reply:
IMHO useful to avoid parsing UTF-8 from UCS-2 ourselves 😓 
https://mathiasbynens.be/notes/javascript-encoding